demo_endtoend_imagenet.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    model = load_pretrained_model(modelpath, SCIVISION_YAML_CONFIG=scivision_conf, allow_install=True)
</pre><p>I suggest we have an explicit parameter for the config file here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/scivision/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
this means we should replace `SCIVISION_YAML_CONFIG ` to `config_filename`?

Reply:
or what do you mean by explicit parameter?

Reply:
> this means we should replace SCIVISION_YAML_CONFIG to config_filename?

Exactly!